File: app/src/main/java/huce/fit/appreadstories/account/register/AccountRegisterPresenter.java
package huce.fit.appreadstories.account.register;

import huce.fit.appreadstories.account.BaseAccountPresenter;

public interface AccountRegisterPresenter extends BaseAccountPresenter {

    void register(String username, String password, String email, String name, String birthday);
    void openDatePicker();
}

File: app/src/main/java/huce/fit/appreadstories/account/register/AccountRegisterPresenter.kt
package huce.fit.appreadstories.account.register

import huce.fit.appreadstories.account.BaseAccountPresenter

interface AccountRegisterPresenter : BaseAccountPresenter {

    fun register(username: String, password: String, email: String, name: String, birthday: String)
    fun openDatePicker()
}
